### ISSUE TITLE
Unhandled exception when GPIO module doesn't exists

### ISSUE THREAD

**Issue by laurentalacoque** [laurentalacoque/TouchSelfie#20]
```
Traceback (most recent call last):
  File "user_interface.py", line 607, in <module>
    ui = UserInterface(config,window_size=(SCREEN_W, SCREEN_H))
  File "user_interface.py", line 145, in __init__
    self.buttons = HWB.Buttons( buttons_pins = HARDWARE_BUTTONS['button_pins'],
mode = HARDWARE_BUTTONS["pull_up_down"], active_state = HARDWARE_BUTTONS["active
_state"])
  File "C:\Users\LA203179\Documents\prog\TouchSelfie\scripts\hardware_buttons.py
", line 42, in __init__
    self.mode = GPIO.PUD_DOWN
NameError: global name 'GPIO' is not defined
Exception AttributeError: "Buttons instance has no attribute '_has_buttons'" in
<bound method Buttons.__del__ of <hardware_buttons.Buttons instance at 0x0000000
0027524C8>> ignored
```

**Comment by laurentalacoque** [comment]
Caused by mode testing outside the safe "RPI_GPIO_EXISTS" clause.
Fixed in 'documentation' branch

**Comment by laurentalacoque** [comment]
fixed in #21
